app/templates/layout.html

```diff
@@ -37,7 +37,8 @@
   {% endfor %}
   {% block body %}{% endblock %}
 </div>
-<div id="footer">
+<div style="width:800px; margin:0 auto" id="footer">
```

Comment:
Consider always mobile first view, use Chrome  Dev tools to view on smaller sizes 


Semantic html



https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer

Reply:
Fixed using semantic html attribute - align="center"